otherlibs/unix/unixLabels.mli

```diff
@@ -192,10 +192,11 @@ type process_status = Unix.process_status =
            the argument is the return code. *)
   | WSIGNALED of int
         (** The process was killed by a signal;
-           the argument is the signal number. *)
+           the argument is the `standard signal number'.
+           See below for more information. *)
   | WSTOPPED of int
         (** The process was stopped by a signal; the argument is the
-           signal number. *)
+           `standard signal number'. See below for more information. *)
 (** The termination status of a process.  See module {!Sys} for the
     definitions of the standard signal numbers.  Note that they are
     not the numbers used by the OS. *)
```

Comment:
Then:

```suggestion
(** The termination status of a process. {b Note that they are
    not the numbers used by the OS}. See module {!Sys} for the
    definitions of the standard signal numbers.   *)
```

Reply:
We could also link directly to the list of signal numbers by adding a label to `Sys`:
```
(** {2:signalnumbers Signal numbers for the standard POSIX signals.} *)
```
and referencing it directly
```
See the  {{!Sys.signalnumbers}signal number section} in module {!Sys} for the ...
```
or maybe directly
```
See module {{!Sys.signalnumbers}Sys} for the ...
```